v2/goi18n/merge_command.go

```diff
@@ -125,12 +125,16 @@ func merge(messageFiles map[string][]byte, sourceLanguageTag language.Tag, outdi
 		}
 		templates := map[string]*i18n.MessageTemplate{}
 		for _, m := range mf.Messages {
-			templates[m.ID] = i18n.NewMessageTemplate(m)
+			template := i18n.NewMessageTemplate(m)
+			if template == nil {
+				return nil, fmt.Errorf("missing translation for message id \"%s\"", m.ID)
+			}
+			templates[m.ID] = template
 		}
 		if mf.Tag == sourceLanguageTag {
 			for _, template := range templates {
 				if sourceMessageTemplates[template.ID] != nil {
-					return nil, fmt.Errorf("multiple source translations for id %s", template.ID)
+					return nil, fmt.Errorf("multiple source translations for id \"%s\"", template.ID)
```

Comment:
```suggestion
					return nil, fmt.Errorf("multiple source translations for id %q", template.ID)
```

Reply:
> Thanks for filing an issue and proposing a fix! This diff proposes treating the missing translation as an error but it isn’t obvious to me that it should be. My instinct would be to just skip over the missing one.

I had the same incline to simply continue. But I also wanted to report the missing translation. My initial implementation was like this:
```golang
template := i18n.NewMessageTemplate(m)
if template == nil {
	fmt.Fprintf(os.Stderr, "missing translation for message id %q\n", m.ID)
	continue
}
templates[m.ID] = template
```

Shall I change the commit to the above mentioned code? I'll update the unittest change accordingly.

P.S.: thanks for `%q`, didn't know that one yet